debezium-core/src/main/java/io/debezium/pipeline/EventDispatcher.java

```diff
@@ -142,7 +142,8 @@ public void changeRecord(DataCollectionSchema schema, Operation operation, Objec
 
     public void dispatchSchemaChangeEvent(T dataCollectionId, SchemaChangeEventEmitter schemaChangeEventEmitter) throws InterruptedException {
         if(!filter.isIncluded(dataCollectionId)) {
-            LOGGER.trace("Skipping data change event for {}", dataCollectionId);
+            LOGGER.trace("Filtering data change event for {}", dataCollectionId);
+            eventListener.onFilteredEvent("source = " + dataCollectionId);
```

Comment:
Unsure if it's appropriate to increment the filtered one here or not, it didn't increment the skipped metric before.

Reply:
The log message was not correct here -it should be `schema` not `data` change event.
I think schema events should not count against the metric - @gunnarmorling Your opinion?